init.sh

```diff
@@ -0,0 +1,13 @@
+if [ ! -d $HOME/.oh-my-zsh ]
+then 
+	echo "Installing oh-my-zsh..."
+	sh -c "$(curl -fsSL https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"
+else 
+echo "oh-my-zsh was already installed"
+fi
+ln -s .dotfiles/.vimrc $HOME/.vimrc
+ln -s .dotfiles/.zshrc $HOME/.zshrc
+if [ -f $ZSH_CUSTOM/aliases.zsh ]; then; rm $ZSH_CUSTOM/aliases.zsh;
```

Comment:
This is not completely valid syntax, since you don't close the if structure with `fi`. You should probably also split it into several lines to make it a bit more readable. Then you don't need the `;` anymore (as this is interpreted as newlines when you write your command on single lines).